resources/providersettings.xml

```diff
@@ -45,7 +45,7 @@
             <dependency type="enable">
               <or>
                 <condition on="setting" setting="plex.authentication" operator="is">0</condition>  <!-- SETTINGS_PROVIDER_AUTHENTICATION_OPTION_LOCAL -->
-                <condition on="setting" setting="plex.token" operator="!is"></condition> 
+                <condition on="setting" setting="plex.token" operator="!is"></condition>
```

Comment:
This change looks unrelated. Can you move it into a separate "cosmetics" commit?